test/ssh-connection/kafka-source-after-ssh-failure-restart-replica.td

```diff
@@ -0,0 +1,34 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# This test script runs after the SSH bastion host has been terminated.
+
+> DROP CLUSTER REPLICA sc.r1
+> CREATE CLUSTER REPLICA sc.r1 SIZE '32'
+
+# TODO(guswynn): these should be correctly namespaced as ssh errors
```

Comment:
Is this TODO intended to be addressed within this PR? If not, what about creating an issue and referencing it here?

Reply:
No, its too much for 1 pr so it will be fixed in a later pr! i will add it to the tracking issue for this feature!